R/shapley.R

```diff
@@ -16,10 +20,10 @@ shapley_weights <- function(m, N, s, weight_zero_m = 10^6) {
 #' Calculate weighted matrix
 #'
 #' @param X data.table
+#' @param use_shapley_weights_in_W Logical
```

Comment:
We should probably do this in a different PR, but to me, it seems that we don't need this argument after we remove the reduce_dim argument. We should then always just use the shapley weights without as the w-vector (i.e. without multiplying with the no column), as the only situation where no can be different from 1 is when reduce_dim = FALSE.

Reply:
Anyway, for now, I suggest
```suggestion
#' @param use_shapley_weights_in_W Logical. Whether to use direct formula based (TRUE) or sampling based (FALSE) weights for the combinations.
```